packages/front-end/components/Layout/SidebarLink.tsx

```diff
@@ -127,61 +128,65 @@ const SidebarLink: FC<SidebarLinkProps> = (props) => {
             [styles.open]: open || selected,
           })}
         >
-          {props.subLinks.map((l) => {
-            if (l.superAdmin && !admin) return null;
+          {props.subLinks
+            .filter(
+              (subLink) => !subLink.feature || growthbook.isOn(subLink.feature)
+            )
```

Comment:
Adding this `.filter()` is the change—the rest is prettier auto-formatting the whitespace.